Documentation/git-config.txt

```diff
@@ -253,14 +257,6 @@ $XDG_CONFIG_HOME/git/config::
 $GIT_DIR/config::
 	Repository specific configuration file.
 
-On Windows, as there is no central `/etc/` directory, there is yet another
-config file (located at `$PROGRAMDATA/Git/config`), intended to contain
-settings for *all* Git-related software running on the machine. Consequently,
-this config file takes an even lower precedence than the
-`$(prefix)/etc/gitconfig` file. Typically `$PROGRAMDATA` points to
-`C:\ProgramData` (on Windows XP the equivalent in `$ALLUSERSPROFILE` is used,
-i.e. `C:\Documents and Settings\All Users\Application Data\Git\config`).
-
```

Comment:
This explanation would need to be kept in one form or another, but I agree that with the additions above, it should be reworded because it now adds depth to the information above (as opposed to adding new information that has not been hinted at before).